client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/style.scss

```diff
@@ -73,21 +73,7 @@
 
 }
 
-.is-videopress-tv-stepper,
-.is-videopress-tv-purchase-stepper {
-	font-family: Inter, -apple-system, system-ui, blinkmacsystemfont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
-
-	.processing-step__progress-step {
-		font-family: inherit;
-		font-size: rem(38px); //typography-exception
-		font-weight: 700;
-		letter-spacing: -0.02em;
-	}
-}
-
-.is-videopress-stepper.is-processing-step,
-.is-videopress-tv-stepper,
-.is-videopress-tv-purchase-stepper {
+.is-videopress-stepper.is-processing-step {
```

Comment:
Can we remove this?

Reply:
I'm removing the `videopress` flow in a separate PR so there's less changes to review.